content/docs/setup/behavioral_changes.md

```diff
@@ -0,0 +1,91 @@
+
+### Service Graph is now deprecated in favor of Kiali
+
+The Service Graph component has now been deprecated in favor of the Kiali Monitoring tool.  For more information about Kiali and its visualization capabilities please refer to the [Telemetry section](/docs/tasks/telemetry/) of the documentation and the [Kiali website](https://www.kiali.io/).
+
+If you would like to see new features as they are being developed please check out the [Kiali service mesh observability project](https://www.youtube.com/channel/UCcm2NzDN_UCZKk2yYmOpc5w) on YouTube where you will find the end of sprint demos.
+
+### Egress Gateway is now disabled by default
+
+The optional egress gateway is disabled by default.  It is enabled in the demo profile for users to explore but disabled in all other profiles by default.  If you need to control and secure your outbound traffic through the egress gateway, you will need to enable `gateways.istio-egressgateway.enabled=true` manually in any of the non-demo profiles. 
+
+## Policy & Telemetry Changes
+
+### Policy and quota checks are now disabled by default
+
+Istio-policy is disabled by default.  It is enabled in the demo profile for users to explore but disabled in all other profiles.  This change is only for Istio-policy not for istio-telemetry.
+
+In order to re-enable policy checking, run helm template with `--set global.disablePolicyChecks=false` and re-apply the configuration.
+
+## Security Changes
+
+### RBAC Configuration has been modified to correctly implement cluster scoping
+
+The RbacConfig resource has been replaced with the ClusterRbacConfig resource.   Refer to the [Migrating RbacConfig to ClusterRbacConfig](/docs/setup/kubernetes/upgrade/#migrating-from-rbacconfig-to-clusterrbacconfig) documentation for migration instructions.
+
+### Retrieve TLS key pairs via SecretDiscoveryService (SDS) Alpha
+
+Envoy can now optionally retrieve X509 private key and certificate pairs via Envoy's new SecretDiscoveryService (SDS) API instead of a secret volume mount.    Refer to the [SDS Tutorial](/docs/tasks/security/auth-sds/) for more details and note well that this *alpha* feature does not support non-disruptive upgrades of production clusters.
+
+You can also optionally configure a TLS or mutual TLS ingress gateway to fetch X509 certificate and key pairs using SDS.   Refer to the [SDS for Ingress Gateways Tutorial](/docs/tasks/traffic-management/secure-ingress/sds/) for more details.
+
+### Citadel agents now supports Vault Certificate Authorities
+
+Istio users can now optionally "Bring their own CA" by integrating Vault CAs into Istio.  Refer to the [Vault CA Integration](/docs/tasks/security/vault-ca/) documentation for more more details.
+
+### Configurable organization and cluster-specific trust domains
+
+The trust domain defaults to “cluster.local”.  Refer to (**TODO LINK**) for more details.
+
+### RBAC Authorization for TCP
```

Comment:
No need to say "RBAC". "Authorization for TCP" is enough.

Reply:
no longer relevant.  I removed this section since it's redundant with the release notes